workspaces/resource-optimization/plugins/resource-optimization/README.md

```diff
@@ -1,13 +1,158 @@
+
+export const Root = ({ children }: PropsWithChildren<{}>) => (
+  <SidebarPage>
+    <Sidebar>
+      <SidebarLogo />
+      <SidebarGroup label="Search" icon={<SearchIcon />} to="/search">
+        <SidebarSearchModal />
+      </SidebarGroup>
+      <SidebarDivider />
+      <SidebarGroup label="Menu" icon={<MenuIcon />}>
+        {/* Global nav, not org-specific */}
+        <SidebarItem icon={HomeIcon} to="catalog" text="Home" />
+        <SidebarItem icon={ExtensionIcon} to="api-docs" text="APIs" />
+        <SidebarItem icon={LibraryBooks} to="docs" text="Docs" />
+        <SidebarItem icon={CreateComponentIcon} to="create" text="Create..." />
+        {/* End global nav */}
+        <SidebarDivider />
+        <SidebarScrollWrapper>
+          <SidebarItem icon={MapIcon} to="tech-radar" text="Tech Radar" />
+        </SidebarScrollWrapper>
++       <SidebarItem
++         icon={ResourceOptimizationIconOutlined}
++         to="/resource-optimization"
++         text="Optimizations"
++       />
+      </SidebarGroup>
+      <SidebarSpace />
+      <SidebarDivider />
+      <SidebarGroup
+        label="Settings"
+        icon={<UserSettingsSignInAvatar />}
+        to="/settings"
+      >
+        <SidebarSettings />
+      </SidebarGroup>
+    </Sidebar>
+    {children}
+  </SidebarPage>
+);
+```
+
+#### Option 2: Dynamic plugin
+
+This option describes how to install this plugin in Red Hat Developer Hub (RHDH).
+It is advised to review the official documentation corresponding to your deployment.
+
+1. Open Red Hat Developer Hub:
+2. Navigate to the Helm Chart:
+   - Go to the Helm tab in your **Red Hat OpenShift console**.
+3. Edit the Helm Chart Values:
+   - Include the plugin in the Helm chart values as shown in the example below:
+
+```yaml
+global:
+  dynamic:
+    plugins:
+      - package: '@backstage-community/plugin-resource-optimization-dynamic@1.0.0'
+        integrity: 'sha512-Qd8pniy1yRx+x7LnwjzQ6k9zP+C1yex24MaCcx7dGDPT/XbTokwoSZr4baSSn8jUA6P45NUUevu1d629mG4JGQ=='
+      - package: '@backstage-community/plugin-resource-optimization-backend-dynamic@1.0.0'
+        integrity: 'sha512-Q1d629mG4JGd8pniy1yRx+x7LnwjzQ6k9zP+C1yex24MaCcx7dGDPT/XbTokwoSZr4baSSn8jUA6P45NUUevud8pniy1yRQx=='
+```
+
+4. Include the below configuration in the `app-config.yaml` ConfigMap to setup the front-end side of the plugin
+
+```yaml
+dynamicPlugins:
+  frontend:
+    backstage-community.plugin-resource-optimization:
+      appIcons:
+        - name: resourceOptimizationIconOutlined
+          importName: ResourceOptimizationIconOutlined
+      routeBindings:
+        targets:
+          - name: resourceOptimizationPlugin
+      dynamicRoutes:
+        - path: /resource-optimization
+          importName: ResourceOptimizationPage
+          menuItem:
+            icon: resourceOptimizationIconOutlined
+            text: Optimizations
+```
+
+### Configuration
+
+#### Step 1: Create a service account in Red Hat Hybrid Cloud Console (RHHCC)
```

Comment:
@jkilzi just FYI, mostly I have seen only HCC been used for referring to Red Hat Hybrid Cloud Console but this is okay too.